netshare/api.py

```diff
@@ -0,0 +1,41 @@
+from typing import Optional
+
+import netshare.utils.ray as ray
+from netshare.configs import change_work_folder, load_from_file
+from netshare.generate.generate import generate
+from netshare.input_adapters.adapt_input import input_adapter
+from netshare.learn.learn import learn
+from netshare.output_adapters.dashboard import visualize_api
+from netshare.output_adapters.output_adapter import (
+    get_output_data_folder,
+    output_adapter,
+)
+from netshare.utils.paths import get_preprocessed_data_folder, get_visualization_folder
+
+
+class Generator(object):
+    def __init__(self, config: str, work_folder: Optional[str] = None):
+        load_from_file(config)
+        change_work_folder(work_folder)
+        ray.init(address="auto")
+
+    def train(self) -> None:
+        input_adapter()
```

Comment:
Nitpicks: `generate` and `learn` are verbs, but `input_adapter`, `output_adapter` are nouns. They are just different steps in the pipeline, so maybe it is better to use the same form of words to name them to reduce confusion

Reply:
You're absolutely right!
Love the idea!
Will do :)